main.tf

```diff
@@ -68,15 +68,15 @@ data "aws_iam_policy_document" "state_policy_root" {
 data "aws_iam_policy_document" "state_bucket_policy" {
   dynamic "statement" {
     for_each = flatten([
-      for policy in concat(var.state_policies, local.bukcet_policies) : policy.statement
+      for policy in concat(var.state_policies, local.bucket_policies) : policy.statement
     ])
 
     content {
       sid         = lookup(statement.value, "sid", null)
       effect      = lookup(statement.value, "effect", null)
       actions     = lookup(statement.value, "actions", null)
       not_actions = lookup(statement.value, "not_actions", null)
-      resources   = [local.bucket_arn]
+      resources   = [local.bucket_arn, "${local.bucket_arn}/*"]
```

Comment:
update code style please to be uniform, use multiline and format (like elsewhere in the code)

Reply:
could you check whether it's aligned properly now?